src/app/controllers/inventories.py

```diff
@@ -32,3 +36,35 @@ def list_all_requirements():
         }
 
     return jsonify(return_dados), 200
+
+@inventory.route("/create", methods= ["POST"])
+@requires_access_level("WRITE")
+def create():
+    list_keys = ["product_category_id", "product_code", "title", "value", "brand", "template", "description"]
+    
+    data = allkeys_in(request.get_json(), list_keys)
+    
+    if exist_product_code('data', 'product_code'):
+        return jsonify({"error": "Esse código de produto já existe"}), 400
+    
+    if data["value"] <= 0:
+        return jsonify({"error": "O valor não pode ser menor ou igual a zero"}), 400
+    
```

Comment:
If data["user_id"] vai dar key error, porque essa chave não existe. Substituir por:

`if "user_id" not in data.keys():`